docs/develop/develop-overview.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Develop Office Add-ins
+description: An introduction to developing Office Add-ins.
+ms.date: 10/28/2019
+localization_priority: Priority
+---
+
+# Develop Office Add-ins
+
+Office Add-ins can extend the UI and functionality of Office applications and interact with content in Office documents. You can use familiar web technologies to create Office Add-ins that extend and interact with Word, Excel, PowerPoint, OneNote, Project, or Outlook, and your solution can run in Office across multiple platforms, including Windows, Mac, iPad, and in a browser. This article provides an introduction to developing Office Add-ins.
+
+> [!TIP]
+> If you haven't already done so, please review [Office Add-ins platform overview](../overview/office-add-ins.md) for information that sets context for the topics covered in this article.
+
+## Core development concepts 
+
+As described in [Office Add-ins platform overview](../overview/office-add-ins.md), an Office Add-in consists of two parts:
```

Comment:
It feels weird to have this link right after a section telling you to go to that link. Maybe remove the lead-in on this sentence and start with "An Office Add-in consists..."